crates/ln-dlc-node/src/channel.rs

```diff
@@ -148,7 +128,8 @@ impl Channel {
                     channel_details.inbound_capacity_msat / 1000,
                     0,
                     channel_details.counterparty.node_id,
-                    None,
+                    // legacy record
+                    1,
```

Comment:
:thinking: I think for some channels having a liquidity option ID makes no sense e.g. public channels.

An `Option` might be a better choice in terms of modelling.

Reply:
Agreed